docs/schemaform/form-config.md

```diff
@@ -180,22 +180,29 @@ We've also been adding some additional uiSchema functionality not found in the r
   'ui:required': function (formData) {
     return true || false;
   },
-  
+
   // This is an array of validation functions or objects that can be used to add validation
   // that is not possible through JSON Schema. See below for the properties passed
   // to the validation functions and how to use them.
   'ui:validations': [
-    function (errors, fieldData, pageData, fieldSchema, errorMessages) {
+    /**
+     * Note the difference between the three data parameters:
+     *
+     * @param {any} fieldData The data for the current field being validated
+     * @param {object} pageData The data for all the fields on the current page
+     * @param {object} formData The data for all the fields in every page
+     */
+    function (errors, fieldData, pageData, formData fieldSchema, errorMessages) {
```

Comment:
Ha, yeah took me a second to realize that you weren't obsessively going over documentation and removing unneeded whitespace :)

Isn't the formData also flattened whereas the pageData maintains the form config structure? May want to explain that if so.

Looks like you're missing a comma after formData